CHANGES.md

```diff
@@ -1,5 +1,43 @@
 # Changes
 
+## edge-21.4.4
+
+This edge release further consolidates the control plane by removing the
+`linkerd-controller` deployment and moving the `sp-validator` container into the
+`destination` deployment.
+
+Annotation inheritance has been added so that +all* Linkerd annotations
+on a `Namespace` resource will be inherited by pods within that namespace.
+In addition, the `config.linkerd.io/proxy-await` annotation has been added which
+enables the [linkerd-await](https://github.com/olix0r/linkerd-await)
+functionality, simplifying the implementation of the await behavior.
+
+Some of the `linkerd check` functionality has been updated. The command
+ensures that annotations and labels are properly located in the YAML and
+displays hinit anchors for core health checks.
```

Comment:
What are you referring to here about the hint anchors? There was a bug related to anchors, but that got fixed in 2.10.1

Reply:
That note came from #6023 

If it's the same thing that is in 2.10.1 and it doesn't need to be included, I'll remove it

Reply:
Yep, that made it into 2.10.1